src/App.jsx

```diff
@@ -14,45 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'alphabetically';
+const SORT_LENGTH = 'length';
+
+function getPreparedGood(goods, { sortField, isReverse }) {
+  const prepeardeGoods = [...goods];
+
+  if (sortField) {
+    prepeardeGoods.sort((item1, item2) => {
+      switch (sortField) {
+        case SORT_ALPHABETICALLY:
+          return item1.localeCompare(item2);
+        case SORT_LENGTH:
+          return item1.length - item2.length;
+        default:
+          return 0;
```

Comment:
In that case, the sort method will do nothing. Consider moving switch...case outside of the sort callback to avoid extra calls